.github/workflows/docker-build-and-release.yml

```diff
@@ -0,0 +1,47 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+#  pull_request: # only for demonstration in the pr
```

Comment:
commenting this in would create a container on every pr. Which is sometimes helpful - but creates a lot of containers.